### PR TITLE
build(Now): Add now build for PR preview

### PR DESCRIPTION
**Pull request by martahj**
Adds capability for creating static builds using Zeit's now, which
will enable PRs to include a preview link to the updated
documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,18 +6,21 @@
...
-7. Pages are mapped to routes for our documentation
-8. Open a PR
-9. Profit :D
+6. Add the components to pages in /pages (Pages are mapped to routes for our documentation)
+7. Install [now](https://zeit.co/now#whats-now) if you do not already have it installed.
+8. Run `yarn now-build` to get a shareable link to the updated docs copied to your
+clipboard.
+9. Open a PR, making sure to include the link to the updated docs so others can
+see your wonderful changes.
+10. Profit :D
```
Comment (kristajg) [inline README.md]: Should be actual emoji

Comment (brad-decker) [reply README.md]: :trollface:

**Comment by kristajg** [comment]
Quick question - where are these cloud deploys hosted @brad-decker

**Comment by brad-decker** [comment]
https://zeit.co/now @kristajg

**Pull request merged**